public/locales/en/translation.json

```diff
@@ -13,8 +14,34 @@
     },
     "parentOne": "Parent link"
   },
+  "secondaryLink": [],
```

Comment:
Should this array include the link names on lines 19 and 20?

Reply:
The empty array is there on purpose, it was a placeholder. The array is now filled in the next PR.